docs/partiql-tests-schema-proposal.md

```diff
@@ -205,12 +206,76 @@ $time::'02:30:59'
 
 ---
 
+#### PartiQL Evaluation Modes
+The PartiQL specification defines two dynamic type mismatching evaluation modes (i.e. when a PartiQL statement is run 
+without schema). As defined in the PartiQL specification, these modes are:
+- Permissive mode -- dynamic typing mismatches are neglected and PartiQL returns `MISSING`
+- Type checking mode -- dynamic type mismatches result in evaluation errors
+
+The naming of these modes can be somewhat confusing especially "type checking mode", which is sometimes referred to as
+`STRICT` mode in the specification and Kotlin reference implementation. For the purposes of this document and the
+conformance tests, we will refer to permissive mode as `EvalModeCoerce` and type checking mode as `EvalModeError`. 
+These names can be changed in the future once we improve the terminology in the specification.
```

Comment:
Does it makes sense to have a TODO issue in `partiql-docs` for any required spec. change (like a spike) and linking it here?

Reply:
Yeah, I think it makes sense to add. Created https://github.com/partiql/partiql-docs/issues/24 to track and added a link.